requirements/base.txt

```diff
@@ -12,12 +12,12 @@ django-cors-headers==3.2.1  # via -r requirements/base.in
 django-haystack==2.8.0    # via -r requirements/base.in
 django-rest-swagger==2.1.2  # via -r requirements/base.in
 django-waffle==0.20.0     # via edx-django-utils, edx-drf-extensions
-django==1.11.29           # via -r requirements/base.in, django-cors-headers, django-haystack, djangorestframework, drf-jwt, edx-django-release-util, edx-django-utils, edx-drf-extensions, rest-condition
+django==2.2.11            # via -r requirements/base.in, django-cors-headers, django-haystack, djangorestframework, drf-jwt, edx-django-release-util, edx-django-utils, edx-drf-extensions, rest-condition
 djangorestframework==3.11.0  # via -r requirements/base.in, django-rest-swagger, drf-jwt, edx-drf-extensions, rest-condition
 drf-jwt==1.14.0           # via -c requirements/constraints.txt, edx-drf-extensions
```

Comment:
I think you can remove the constraint now by making the changes similar to https://github.com/edx/portal-designer/pull/84. It should be if it can be done in the same PR similar to other repos.